src/test-utilities/legacy.tsx

```diff
@@ -133,9 +102,10 @@ export function mountWithAppProvider<P>(
 
   const link = new Link(ctx.link);
 
-  const appBridge = ctx.appBridge || null;
+  // Yes this is bad typing, we'll fix it later
```

Comment:
I don't think we'll need to go and fix this later. It's a little annoying but it doesn't cause us any harm and there isn't to much value going back and changing it since it's legacy now and we're moving away from `enzyme`.